pytorch_lightning/plugins/native_amp.py

```diff
@@ -38,6 +38,11 @@ def backward(self, closure_loss, optimizer, opt_idx, *args, **kwargs):
 
         # once backward has been applied, release graph
         closure_loss = closure_loss.detach()
+
+        # unscale gradient to allow analyze within `on_after_backward`
+        if not self.trainer.should_accumulate():
```

Comment:
trainer does not have this method, how is it possible that test passes?

Reply:
oh.. as @ydcjeff says, you are skipping drone which is the only place where we run amp test :)

Reply:
Yes, doing in #4393